### PR TITLE
Wrong file names in Zenodo upload data bit

### PR DESCRIPTION
**Pull request by nomadscientist**
Please help! Links have a typo!

### CODE REVIEW

File: topics/transcriptomics/tutorials/droplet-quantification-preprocessing/tutorial.md [hunk 1]
```diff
@@ -502,7 +502,13 @@ You should now have `111` barcodes! You now have an annotated expression matrix
 >
 {: .hands_on}
 
+<<<<<<< Updated upstream
 {% icon congratulations %} Congrats! Your object is ready to for the scanpy pipeline! However, it may be that you want to combine this object with others like it, for instance, maybe you ran 5 samples, and you are starting with 10 FASTQ files...
+=======
```
Comment (mtekman) [inline topics/transcriptomics/tutorials/droplet-quantification-preprocessing/tutorial.md:507]:
```suggestion
```

File: topics/transcriptomics/tutorials/droplet-quantification-preprocessing/tutorial.md [hunk 2]
```diff
@@ -502,7 +502,13 @@ You should now have `111` barcodes! You now have an annotated expression matrix
...
+{% icon congratulations %} Congrats! Your object is ready to for the scanpy pipeline! You can can check your work against the [example history](https://humancellatlas.usegalaxy.eu/u/wendi.bacon.training/h/pre-processing-with-alevin---part-1---answer-key).
+
+However, it may be that you want to combine this object with others like it, for instance, maybe you ran 5 samples, and you are starting with 10 FASTQ files...
+>>>>>>> Stashed changes
```
Comment (mtekman) [inline topics/transcriptomics/tutorials/droplet-quantification-preprocessing/tutorial.md:511]:
```suggestion
```

**Review by hexylena — DISMISSED** [review]

**Comment by hexylena** [comment]
Still a merge conflict

**Comment by nomadscientist** [comment]
Is that merge conflict fixed now? I think?

**Review by hexylena — APPROVED** [review]
Yep!

**Pull request merged**